### ISSUE TITLE
Uninstall prompts for a version when there is only one installed

### ISSUE THREAD

**Issue by christianrondeau** [chocolatey/choco#186]
I'm not sure how to reproduce. Maybe this happens when failing previous uninstalls.

Here are examples:

```bash
PS C:\> choco uninstall vagrant
Chocolatey v0.9.9.2
Uninstalling the following packages:
vagrant
Which version of vagrant would you like to uninstall?
 1) None [Default - Press Enter]
 2) 1.5.4
```

Another example:

```bash
PS C:\> choco uninstall virtualbox
Chocolatey v0.9.9.2
Uninstalling the following packages:
virtualbox
Which version of virtualbox would you like to uninstall?
 1) None [Default - Press Enter]
 2) 4.3.12
```

Here is an example of the previous attempt to install all versions of a package. Might be related.

```bash
PS C:\> choco uninstall vagrant
Chocolatey v0.9.9.2
Uninstalling the following packages:
vagrant
Which version of vagrant would you like to uninstall?
 1) None [Default - Press Enter]
 2) 1.5.4
 3) 1.6.3
 4) 1.6.5
 5) All versions
5
You selected to remove all versions of vagrant
vagrant not uninstalled. An error occurred during uninstall:
 Could not find a part of the path 'C:\ProgramData\chocolatey\lib\vagrant.1.5.4\vagrant.1.5.4.nupkg'.
This is try 1/3. Retrying after 300 milliseconds.
 Error converted to warning:
 Cannot create a file when that file already exists.

This is try 2/3. Retrying after 400 milliseconds.
 Error converted to warning:
 Cannot create a file when that file already exists.

Maximum tries of 3 reached. Throwing error.
Move failed with message:
 Cannot create a file when that file already exists.

 Attempting backup move method.

vagrant v1.6.3
 vagrant has been successfully uninstalled.
This is try 1/3. Retrying after 300 milliseconds.
 Error converted to warning:
 Cannot create a file when that file already exists.

This is try 2/3. Retrying after 400 milliseconds.
 Error converted to warning:
 Cannot create a file when that file already exists.

Maximum tries of 3 reached. Throwing error.
Move failed with message:
 Cannot create a file when that file already exists.

 Attempting backup move method.

vagrant v1.6.5
 Skipping auto uninstaller - AutoUninstaller feature is not enabled.
 Skipping auto uninstaller - AutoUninstaller feature is not enabled.
 vagrant has been successfully uninstalled.

Chocolatey uninstalled 2/3 packages. 1 packages failed.
See the log for details.
Failures
 - vagrant
```

For more details: https://groups.google.com/forum/#!topic/chocolatey/UVncL7PxXRg

(Moved from https://github.com/chocolatey/chocolatey/issues/694)

**Comment by ferventcoder** [comment]
Thanks for reporting this. Should be resolved in 0.9.9.5! :)

**Comment by christianrondeau** [comment]
Great! :)
